### DISCUSSION TITLE
how i post data & output response code?

### DISCUSSION THREAD

**Discussion by secghost** [curl/curl#9869] [Q&A] [answered]
i need post some data & get the response code like 200 but i cant & error

$ curl -I --data "data1=data1&data2=data2&data3=data3" https://localhost/login
Warning: You can only select one HTTP request method! You asked for both POST
Warning: (-d, --data) and HEAD (-I, --head).

**Comment by bagder** [comment, marked as answer]
`-I` sends a `HEAD` request, `--data` sends a `POST` request. You can't do both.

You probably want `-i` instead of `-I` to include the response headers in the output, or `-D` to save the response headers in a separate file or just `-v` (for verbose) to get to see some header traffic.

**Reply by secghost** [reply]
so what the command to that
